LAKMobile/src/components/NoJobs.tsx

```diff
@@ -0,0 +1,114 @@
+import React from 'react';
+import { StyleSheet } from 'react-native';
+import { View } from 'react-native';
+import { AppButton } from './AppButton';
+import { AppText } from './AppText';
+import { COLORS } from '../../constants';
+import { IconButtonWrapper } from '../components';
+import { NoAvailableJobsIcon, NoJobsIcon, NoMatchingJobsIcon, PlusSignIcon } from '../icons';
+import { ProfileButtonIcon } from '../icons/ProfileButtonIcon';
+
+interface NoJobProps {
+  // type?: 'noAvailableJobs' | 'noJobs' | 'noMatchingJobs';
+  title: string;
+  body: string;
+  buttonName?: string;
+  buttonVisible: boolean;
+  onButtonClick: () => void;
+  iconType?: React.ReactElement;
+  errorImageType: React.ReactElement;
+}
+
+export const NoJobs = ({
+  title,
+  body,
+  buttonName,
+  buttonVisible,
+  onButtonClick,
+  iconType,
```

Comment:
It seems that `iconType` doesn't really describe what this variable is doing exactly. Can this be renamed to `icon`?